docs/getting-started/community-plugins.md

```diff
@@ -85,12 +85,6 @@ Integrate Auth0's robust authentication solutions into your service with the [Au
 [ESLint](https://github.com/amplication/plugins/tree/master/plugins/linter-eslint) is a library that helps find issues and enforce code style in your JavaScript code.
 ESLint has a wide array of rules that can be configured to enforce code quality and consistency.
 
-### Redis
```

Comment:
why did we remove the redis cache plugin? this isn't the same as the Redis Broker plugin that appears below

Reply:
I had two version of the same plugin in here. Redis Caching was below the broker, but I just moved it above.